pdf/model/writer.go

```diff
@@ -112,9 +185,29 @@ func NewPdfWriter() PdfWriter {
 
 	// Creation info.
 	infoDict := core.MakeDict()
-	infoDict.Set("Producer", core.MakeString(getPdfProducer()))
-	infoDict.Set("Creator", core.MakeString(getPdfCreator()))
-	infoObj := core.PdfIndirectObject{}
+	metadata := map[core.PdfObjectName]string{
```

Comment:
Should not use a map here. The order of the keys in the map is not guaranteed by the standard. This is the reason the build fails. The metadata values are potentially written in a different order on each run.

Reply:
Changed this so the order is known and not random anymore